<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vtn_principal</class>
 <widget class="QMainWindow" name="vtn_principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>879</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VENTANA PRINCIPAL</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="txt_nombre">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>90</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_apellido">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>150</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_nombre">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>100</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_apellido">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>160</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Apellido:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_grabar">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>220</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Grabar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="mnb_menu_principal_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>879</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="stb_estado"/>
 </widget>
 <resources/>
 <connections/>
</ui>
